src/storage/blockchain/chainid.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ * Object containing chain IDs for various blockchains and networks.
+ * @type { [key: string]: number }
+ */
+export const CHAIN_IDS: { [key: string]: number } = {
+  eth: 1,
+  'eth:main': 1,
+  'eth:goerli': 5,
+  polygon: 137,
+  'polygon:main': 137,
+  'polygon:mumbai': 80001
+};
+
+/**
+ * Registers a new chain ID for a blockchain and network combination.
+ * If the blockchain or network is not provided, an error will be thrown.
+ * @param {string} blockchain - The blockchain name.
+ * @param {string} network - The network name.
+ * @param {number} chainId - The chain ID to be registered.
+ * @throws {Error} Throws an error if the blockchain name is not provided.
+ * @example
+ * RegisterChainId('eth', 'ropsten', 3);
+ */
+export const RegisterChainId = (blockchain: string, network: string, chainId: number): void => {
```

Comment:
rename RegisterChainId to registerChainId

Reply:
done